<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PremiereFenetre</class>
 <widget class="QDialog" name="PremiereFenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1047</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1047</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>OGC (Outil de Gestion de Contacts)</string>
  </property>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>1041</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Liste des contacts</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableListeContacts">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>1001</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>155</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>155</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date de création</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Entreprise</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Téléphone</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Adresse mail</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeading|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="labelDerniereSuppression">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>291</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dernière suppression de contact : </string>
   </property>
  </widget>
  <widget class="QLabel" name="dateDerniereSuppression">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>680</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelNombreContacts">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>655</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre total de contacts : </string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreContacts">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>655</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NouveauContact">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>665</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nouveau contact</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="champRecherche">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Recherche</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMethodeTri">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Trier par :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choixModeTri">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>85</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Date de création</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ordre alphabétique</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="choixRecherche">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>85</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Recherche par nom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Recherche par entreprise</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Intervalle de dates</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateDebut">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>87</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2026</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFin">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>87</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximumDate">
    <date>
     <year>2026</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>11</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateFin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>et le</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateDebut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher entre le</string>
   </property>
  </widget>
  <widget class="QPushButton" name="consulterContact">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>665</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Consulter contact</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportJson">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>665</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Export JSON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ListeTotalTaches">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>25</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Liste des tâches</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
